docs/_documentations/troubleshooting.md

```diff
@@ -416,7 +416,46 @@ These steps reproduce the issue:
 6. Click **Finish**. The OpenAPI generator fails if the folder doesn't already exist.
 
 **Workaround:**
-- Manually create the output folder before you start the OpenAPI generator wizard. In the wizard, you can manually edit the **Output folder** text field. Ensure that the path points to a valid folder in the project.
-- Or you can use the **Browse...** button to select the folder that you want to use. The **Output folder** field is updated with the path.
+For the VS Code extension, manually create the output folder before you start the OpenAPI generator wizard. In the wizard, you can manually edit the **Output folder** text field. Ensure that the path points to a valid folder in the project.
```

Comment:
Michael, minor change that I didn't catch earlier:
"In the wizard, you can manually edit the **Output folder** text field. Ensure that the path points to a valid folder in the project."
to
"In the wizard, you can also create the **Output folder** in the browse dialog. Ensure that the path points to a valid folder in the project."

Reply:
Modified according to your comments.